packages/trader/src/Stores/Helpers/client-notifications.js

```diff
@@ -262,13 +346,17 @@ const checkAccountStatus = (account_status, client, addNotification, loginid) =>
         ukrts_max_turnover_limit_not_set,
         professional,
     } = getStatusValidations(status);
+
+    const { identity, document } = account_status.authentication;
+    addVerificationNotifications(identity, document, addNotification, unwelcome);
+
     const is_mf_retail = client.landing_company_shortcode === 'maltainvest' && !professional;
 
     if (cashier_locked)        addNotification(clientNotifications.cashier_locked);
     if (withdrawal_locked)     addNotification(clientNotifications.withdrawal_locked);
     if (mt5_withdrawal_locked) addNotification(clientNotifications.mt5_withdrawal_locked);
     if (document_needs_action) addNotification(clientNotifications.document_needs_action);
-    if (unwelcome)             addNotification(clientNotifications.unwelcome);
+    if (unwelcome && !identity && !document) addNotification(clientNotifications.unwelcome);
```

Comment:
unwelcome is deprecated

Reply:
@mustofa-binary i think we still have `unwelcome`?

Reply:
my bad, yes, we still need unwelcome, its prompt_client_to_authenticate is the deprecated status 👍